wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/CellComponents/SenderCellComponent.swift

```diff
@@ -80,6 +80,11 @@ final class SenderCellComponent: UIView {
            let userSession = ZMUserSession.shared() {
             observerToken = UserChangeInfo.add(observer: self, for: user, in: userSession)
         }
+
+        // We need to call that method here to restraint the authorLabel moving
+        // outside of the view and then back to its position. For more information
+        // check the ticket: https://wearezeta.atlassian.net/browse/WPB-1955
+        self.layoutIfNeeded()
```

Comment:
I would think about moving this call into `configureViews` method, I think it fits better there as an immediate action related to views. What do you think?

Reply:
Done!